dz_8/src/main/java/org/example/Student.java

```diff
@@ -0,0 +1,58 @@
+package org.example;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Student {
+    final private int uniqueStudentId;
+    final private String studentName;
+    final private String studentSurname;
+
+
+    public Student(int uniqueStudentId, String studentName, String studentSurname) {
+        this.uniqueStudentId = uniqueStudentId;
+        this.studentName = studentName;
+        this.studentSurname = studentSurname;
+
+    }
+
+     public class StudentsGroup {
```

Comment:
`StudentsGroup` is not belong to `Student` class they should be in separate files